themes/Minimal.bgptheme

```diff
@@ -0,0 +1,61 @@
+##############################################################################
+# Changes the prompt to a Debian-style one that truncates pwd to a max length
+# depending on the terminal column width. Also uses the prompt_callback
+# function of bash-git-prompt to set the window title to almost the same
+# Debian-style. This version has been tweaked for Ubuntu standard terminal 
+# fonts.
+#
+# The prompt will use a Debian-style on the form
+#
+# relative-path-from-git-toplevel-dir bash-git-prompt-info <exit status>
+# HH:MM:SS ▶
+#
+# The window title will have the form
+# relative-path-from-git-toplevel-dir
+#
+# Example usage:
+# if [ -f ~/.bash-git-prompt/gitprompt.sh ]; then
+#   GIT_PROMPT_THEME=Minimal
+#   source ~/.bash-git-prompt/gitprompt.sh
+# fi
+#
+# Imbibinebe <imbibinebe@gmail.com> [https://github.com/imbibinebe]
+##############################################################################
+override_git_prompt_colors() {
+  GIT_PROMPT_THEME_NAME="Minimal"
+
+  #Overrides the prompt_callback function used by bash-git-prompt
+  function prompt_callback {
+    GIT_CONTAINER_FOLDER_FULLPATH=$(git rev-parse --show-toplevel)
```

Comment:
@magicmonty / @imbibinebe 

This line is likely the cause of https://github.com/magicmonty/bash-git-prompt/pull/248#issuecomment-232460229

I don't know the codebase well enough to repair that myself, but I'm pretty sure that the bug would be fixed if you called `git rev-parse --show-toplevel` only when we're inside a git repo directory.

If that is not possible, maybe @magicmonty could provide a conditional function (e.g. `cond-git` that delegates only to git when we're inside a repo). This way, theme-authors could make use of all git commands. In this case `cond-git rev-parse --show-toplevel`.

Maybe @magicmonty could simply provide a variable like `GIT_TOPLEVEL_FOLDER`. 

As a user, I expect the output to be straight to the point in order to be of good use, so I would very much appreciate this theme.